<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminSideClass</class>
 <widget class="QMainWindow" name="AdminSideClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1439</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdminSide</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="backgroundWindow">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>1161</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AutomaticTellerMachine/main.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="LoadWindow">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1161</width>
       <height>701</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/AutomaticTellerMachine/main.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>140</y>
        <width>721</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="LoadInput">
       <item row="0" column="2">
        <widget class="QLineEdit" name="LoadIdInput">
         <property name="font">
          <font>
           <pointsize>36</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="LoadPasswordInput">
         <property name="font">
          <font>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="LoadIdLable">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>账号</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="LoadPasswordLable">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>密码</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="SignButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>470</y>
        <width>261</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
     <widget class="QFrame" name="LoadDashboardWindow">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1161</width>
        <height>701</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/AutomaticTellerMachine/main.png);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>160</y>
         <width>351</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string>资金池现有现金：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>260</y>
         <width>351</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string>时间：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="PoolExistingFunds">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>160</y>
         <width>351</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="LoadDashboardWindowTime">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>260</y>
         <width>351</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="HeadWindow">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Title">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>64</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 255);</string>
     </property>
     <property name="text">
      <string>ATM管理端</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>271</width>
       <height>631</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="Login">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Exit">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="MenuWindow">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>271</width>
       <height>631</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>261</width>
        <height>621</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="UserQueriesButton">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>用户查询</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="UserFreezesButton">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>用户冻结</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="USerUnfreezeButtton">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>用户解冻</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="UserLogsOffButton">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>用户注销</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="FundsTrackingButton">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>资金跟踪</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PoolFillingButton">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>资金池填充</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PoolEmptyButton">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>资金池清空</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LoadExit">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AutomaticTellerMachine.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Exit</sender>
   <signal>clicked()</signal>
   <receiver>AdminSideClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>719</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Login</sender>
   <signal>clicked()</signal>
   <receiver>LoadWindow</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>860</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Exit</sender>
   <signal>clicked()</signal>
   <receiver>LoadWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>860</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LoadExit</sender>
   <signal>clicked()</signal>
   <receiver>LoadDashboardWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>664</y>
    </hint>
    <hint type="destinationlabel">
     <x>860</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
